tests/data/test_history.py

```diff
@@ -14,9 +14,9 @@
 from freqtrade.configuration import TimeRange
 from freqtrade.data import history
 from freqtrade.data.history import (_load_cached_data_for_updating,
```

Comment:
General sidenote (not for this PR).
This is a very inconsistent way to import ... we should either import history and use it's sub-properties - or import each needed value on it's own ...

Reply:
Completelt agree, I also noticed this and even started changing in tests in one of the stashed work...

Reply:
hm - too late - already done ;) see #2675